### PR TITLE
supprot batchnorm1d

### PR DESCRIPTION
**Pull request by FeboReigns**
null

### CODE REVIEW

File: brocolli/converter/pytorch_caffe_parser.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def gen_ir(self):
                 if isinstance(module, nn.Conv2d):
                     layer_data = self.rename_Conv(node, module)
                     self.layers.append(layer_data)
-                elif isinstance(module, nn.BatchNorm2d):
+                elif isinstance(module, (nn.BatchNorm2d,nn.BatchNorm1d)):
```
Comment (inisis) [inline brocolli/converter/pytorch_caffe_parser.py]: (nn.BatchNorm1d, nn.BatchNorm2d) could be better

**Review by inisis — APPROVED** [review]
LGTM

**Comment by inisis** [comment]
LGTM
